source/iface/common-reference.rst

```diff
@@ -12,9 +12,10 @@ Each of the following SYCL runtime classes:
 :ref:`command-accessor`, :ref:`buffer`, :ref:`context`, :ref:`device`,
 :ref:`device_image`, :ref:`event`, :ref:`host_accessor`, 
 :ref:`host_sampled_image_accessor`, :ref:`host_unsampled_image_accessor`, 
-:ref:`kernel`, :ref:`kernel_id`, :ref:`sycl::kernel_bundle <iface-kernel-bundle>`, 
+:ref:`kernel`, :ref:`kernel_id`, :ref:`sycl::kernel_bundle`,
```

Comment:
Why do you say that this link is misleading?  Before your change, the link went to the right place.  Now, there is no link.